src/coreComponents/constitutive/contact/ContactBase.cpp

```diff
@@ -54,6 +51,12 @@ ContactBase::ContactBase( string const & name,
                     "to smooth out highly nonlinear behavior associated with 1/0 in addition to avoiding the "
                     "1/0 error." );
 
+  registerWrapper( viewKeyStruct::displacementJumpThresholdString(), &m_displacementJumpThreshold ).
+    setApplyDefaultValue( std::numeric_limits< real64 >::epsilon() ).
```

Comment:
I agree with the need to use tolerances for the fracture state computation according to the jump.
In the Lagrangian Solver I encountered the same problem and it turned out that, al least in that specific solver, the tolerances have to be problem dependent, i.e., considering the element size, the stiffness of the surrounding FE, and so on ...
I know this is completely beyond the scope of this PR, but perhaps to initialize this threshold to something larger, e.g. 10/100 * _tol_ (with _tol_ the default exit value for the linear solver), would help. Indeed, the linear system is usually solved iteratively up to _tol_ (_tol_ = eps only for a direct solver). Moreover, tol is reached by the residual, while we compare the solution (or derived quantities), thus I would suggest the multiplicative factor of at least 10.
I think setting a proper default initialization will help the end user a lot.